Dnn.AdminExperience/Dnn.PersonaBar.Extensions/Services/RolesController.cs

```diff
@@ -37,7 +37,13 @@ public HttpResponseMessage GetRoles(int groupId, string keyword, int startIndex,
             try
             {
                 int total;
-                var roles = Components.RolesController.Instance.GetRoles(this.PortalSettings, groupId, keyword, out total, startIndex, pageSize).Select(RoleDto.FromRoleInfo);
+                var roles = Components.RolesController.Instance.GetRoles(this.PortalSettings, groupId, keyword, out total, startIndex, pageSize)
+                    .Select(r =>
+                    {
+                        var role = RoleDto.FromRoleInfo(r);
+                        role.LocalizedName = Localization.LocalizeRole(role.Name);

```

Comment:
Should `LocalizedName` be a computed property? Or should this logic be part of `RoleDto.FromRoleInfo`?

Should `RoleInfo` have `LocalizedName` added?

Reply:
Good catch.